public/pages/partner.html

```diff
@@ -113,7 +113,7 @@ <h3>UNHCR </h3>
   <h2>Reach out about a partnership</h2>
   <section>
     <div class="columns">
-      <p>Join the list of organizations that have partnered with Refcode to help realize our vision of connecting refugees and immigrants to software engineering careers. There are many ways that you can partner with us, but we are always open to new ideas, so please get in touch!</p>
+      <p>Join the list of organizations that have partnered with Refcode to help realize our vision of connecting refugees and immigrants to software engineering careers. There are many ways that you can partner with us, by hosting a dip jar at your compnay, or you can even hire a gradute from Refcode!. We are always open to new ideas, so please get in touch!</p>
```

Comment:
`!.`

Reply:
change to:

Join the list of organizations that have partnered with Refcode to help realize our vision of connecting refugees and immigrants to software engineering careers. There are many ways that you can partner with us, but we are always open to new ideas, so please get in touch!

Example ways to get in touch:

<ol>
<li>
<li>...
</ol>

Reply:
`Here are some ways you could get involved:

...`